### DISCUSSION TITLE
Problem accessing the workspace after creating it

### DISCUSSION THREAD

**Discussion by hakujaya** [AntlerVC/firetable#1210] [Support / Q&A] [answered]
I have gone through the setup wizard and finishing it all, i also receive the message:

**Nicely done! 🎉
You’ve created a new project.**
You’re ready to start creating some awesome tables in your project. Check out our live playground and templates for inspiration!

If you need help for your specific use case, join our community or start a discussion with us on GitHub.
How was your setup experience?
---------------------------------

Then, after i click the "Let's Go' button, it loads for a while and then it tells me i dont have access to my own workspace

Please help, thank you

![Screenshot 2023-04-11 230944](https://user-images.githubusercontent.com/85265489/231207262-24bd041d-d974-42cd-8b0a-206efdbc7fe7.png)
![Screenshot 2023-04-11 230737](https://user-images.githubusercontent.com/85265489/231207272-06d5b1c7-c6b1-450d-a952-1d4a4256b9dc.png)

**Comment by HeatherEver** [comment]
i'm also having this exact issue - did you manage to solve it?

**Reply by hakujaya** [reply, marked as answer]
I just solve it, unintentionally, i enabled billing on the google cloud platform for the firebase project and immediately i can access it
